vmanager.cabal

```diff
@@ -25,8 +25,10 @@ source-repository head
 
 library
   exposed-modules:
+      Allocator.Allocator
```

Comment:
I think, we only need to expose `Allocator` module here, because other modules is internals of `Allocator`